test/buildenv_check.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/sh
+
+set -eu
+
+TEST_DIR=$(mktemp -d)
+
+CROSS_COMPILE="${CROSS_COMPILE:-""}"
+
+COMMANDS=" \
+basename \
```

Comment:
Looks good, please indent.

Reply:
Done

Reply:
Actually... this makes us dependent on the whitespace handling of other commands, I don't like it...